tools/robustness-job.sh

```diff
@@ -106,7 +106,11 @@ readonly test_flags="-v -timeout=${test_timeout}\
 
 # Set the make target based on ENGINE_MODE
 ENGINE_MODE="${ENGINE_MODE:-}"
-make_target="robustness-tests"
+if [[ "${ENGINE_MODE}" == "" ]]; then
+    make_target="recovery-tests"
+else
+    make_target="robustness-tests"
+fi
```

Comment:
I assume this outdated. Given the `tools/fault-tolerance-job.sh` script above, there's probably no need for this change here.

Reply:
That's the plan, yes. I am trying to make the pipeline work with the new job.sh. Once that is successful, I will commit the updates and remove the robustness-job.sh.